__tests__/api/__utils__/seeds/users_and_follow_seeds.js

```diff
@@ -1,57 +1,62 @@
 const {v4: uuid} = require('uuid');
 const TestConstants = require('../constant');
+const CryptoUtils = require('../../../../utils/crypto');
 
 const {User, UserFollowUser} = require('../../../../databases/models');
 
 const generateUserAndFollowSeeds = async () => {
   const bulks = [];
   const followers = [];
   const followings = [];
+  const signedUserId = TestConstants.MY_USER_ID;
 
   bulks.push({
-    user_id: TestConstants.MY_ANONYMOUS_USER_ID,
-    human_id: `anon_human_id_${0}`,
+    user_id: signedUserId,
+    human_id: `human_id_${0}`,
     country_code: 'ID',
-    username: `anon_username_${0}`,
+    username: `username_${0}`,
     last_active_at: new Date(),
     status: 'Y',
     created_at: new Date(),
     updated_at: new Date(),
-    real_name: `anon_real_name_${0}`,
+    real_name: `real_name_${0}`,
     profile_pic_path: 'profile_pic_path',
     profile_pic_asset_id: 'profile_pic_asset_id',
     profile_pic_public_id: 'profile_pic_public_id',
     bio: 'bio',
     is_banned: false,
-    is_anonymous: true,
+    is_anonymous: false,
     encrypted: 'encrypted',
     allow_anon_dm: true,
     only_received_dm_from_user_following: false,
     is_backdoor_user: false,
-    verified_status: 'VERIFIED'
+    verified_status: 'VERIFIED',
+    is_karma_unlocked: true
   });
 
+  const anonymousUsername = CryptoUtils.getAnonymousUsername(signedUserId);
   bulks.push({
-    user_id: TestConstants.MY_USER_ID,
-    human_id: `human_id_${0}`,
+    user_id: TestConstants.MY_ANONYMOUS_USER_ID,
+    human_id: anonymousUsername,
     country_code: 'ID',
-    username: `username_${0}`,
+    username: anonymousUsername,
     last_active_at: new Date(),
     status: 'Y',
     created_at: new Date(),
     updated_at: new Date(),
-    real_name: `real_name_${0}`,
+    real_name: `anon_real_name_${0}`,
     profile_pic_path: 'profile_pic_path',
     profile_pic_asset_id: 'profile_pic_asset_id',
     profile_pic_public_id: 'profile_pic_public_id',
     bio: 'bio',
     is_banned: false,
-    is_anonymous: false,
+    is_anonymous: true,
     encrypted: 'encrypted',
     allow_anon_dm: true,
     only_received_dm_from_user_following: false,
     is_backdoor_user: false,
-    verified_status: 'VERIFIED'
+    verified_status: 'VERIFIED',
+    is_karma_unlocked: true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made in this hunk are adding new properties to the user objects being pushed into the `bulks` array. The new properties include `is_karma_unlocked`, which is set to true for both users, and `encrypted`, which is set to 'encrypted'. These changes seem to be consistent with the rest of the code and do not introduce any logical errors or security issues. However, it's important to ensure that these new properties are handled correctly in other parts of the application where user data is processed.
```diff
+     encrypted: 'encrypted',
+     is_karma_unlocked: true
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->